ChangeLog.d/fix-iar-warnings.txt

```diff
@@ -1,2 +1,3 @@
 Bugfix
-   * Fix IAR compiler warnings. Contributed by Glenn Strauss in #3835.
+   * Fix IAR compiler warnings. Contributed by Glenn Strauss in #3835, and
+     reported by Kasimashi in #6924.
```

Comment:
(minor) I think we normally just say

```
   * Fix IAR compiler warnings. Contributed by Glenn Strauss in #3835.
     Fixes #6924.
```

Reply:
It's the other way round, actually: we do, in principle, credit external reporters in changelog entries. Always for security issues, not always for ordinary bugs. We generally don't credit contributors these days: it's understood that the credit is in the author fields in the git log.

Reply:
However, in this case the fix was only given in a comment on an issue, and so Glenn is not the author of the commit, and #6924 is not a security issue, but the reporting of warnings issued by a compiler. So yes "I think we normally just say" maybe should be "In this case I think we should just say"